### PR TITLE
Infobox: IE11-Friendly + Style Update

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
### Description

This PR makes Banner IE11-friendly and updates our styles to match the most recent design. Changes to the component itself reflect pattern-matching with the related Toast and Banner components.

#### FF

Screen Shot 2022-03-11 at 11 10 59 AM
Screen Shot 2022-03-11 at 11 09 25 AM

#### IE11

Screen Shot 2022-03-11 at 11 09 18 AM

### CODE REVIEW

**Review by josesolano-okta — APPROVED** [review]
looks great

File: packages/odyssey-react/src/components/Infobox/Infobox.tsx [hunk 1]
```diff
@@ -99,7 +98,7 @@ export const Infobox = withTheme(
         <span className={styles.icon}>{icon[variant]}</span>
 
         <section className={styles.content}>
-          {heading && <Heading visualLevel="6" children={heading} />}
+          {heading && <h1 className={styles.heading}>{heading}</h1>}
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Infobox/Infobox.tsx:101]: hmmm why can't we use the Heading and its styles here?

Comment (edburyenegren-okta) [reply packages/odyssey-react/src/components/Infobox/Infobox.tsx:101]: The related components - Toast and Banner - use matching styles that weren't available via `Heading`. We also allow for theming those properties in the other components, so I wanted to get this one aligned with their approach.

**Comment by edburyenegren-okta** [comment]
My bad @jeffbelser-okta -- I missed that I didn't have an Eng approval here. I can revert this if need be!

File: packages/odyssey-react/src/components/Infobox/Infobox.theme.ts [hunk 2]
```diff
@@ -14,22 +14,41 @@ import type { ThemeReducer } from "@okta/odyssey-react-theme";
...
   // eslint-disable-next-line @okta/odyssey/no-invalid-theme-properties
   MaxLineLength: theme.FontLineLengthMax,
   PaddingBlock: theme.SpaceScale3,
   PaddingInline: theme.SpaceScale3,
-  MarginBlockEnd: theme.SpaceScale3,
-  BorderRadius: theme.BorderRadiusBase,
-  ColumnGap: theme.SpaceScale2,
 
-  IconSize: theme.FontSizeHeading5,
-  IconLineHeight: theme.FontLineHeightHeading5,
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Infobox/Infobox.theme.ts:26]: this would be a breaking change it looks like too

**Pull request merged**